cmd/config.go

```diff
@@ -215,6 +221,15 @@ func getConsolidatedConfig(gs *state.GlobalState, cliConf Config, runnerOpts lib
 	return conf, nil
 }
 
+func warnOnShortHandOverride(a, b lib.Options, bName string, logger logrus.FieldLogger) {
+	if a.Scenarios != nil &&
+		(b.Duration.Valid || b.Iterations.Valid || b.Stages != nil || b.Scenarios != nil) {
+		logger.Warnf(
+			"%q level configuration overrode scenarios configuration entirely",
```

Comment:
Maybe explicitly mention implications, what do you think?

```suggestion
			"%q level configuration overrode scenarios configuration entirely. It's highly likely, that the test load result will be different from what you expect.",
```

Reply:
This usually isn't the problem - the problem is people expect:
1. to run their scenarios but with a different duration, iterations or VUs, but with all other options the same
2. having some other option stay the same - for example `exec`, `env`, and  also `options` for browser

But making the message a bunch of sentences IMO isn't a great idea. The one we have with docker and stuff about not finding files - it still gets users to post it and ask questions. I only expect this is happening less because of google actually now fidning the results from previous asks.